sale_order_type_dummy_confirmation/README.rst

```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================================
+Sale Order Type Dummy Confirmation Integration
+==============================================
+
+We add dummy confirm option on sale types. If company has 'dummy confirm'
+then all orders will be 'dummy confirm', if not, then we will chec if
+sale type has 'dummy confirm' or not.
+
+Installations
+=============
+
+To install this module, you need to:
+
+#. Just install
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+
+Usage
+=====
+
+To use this module, you need to:
+
+#.To use must be select an "sale order type" who has the configuration
```

Comment:
borrar directamente, ya queda bien explicado arriba, por ahora borrar el item "To use must ..." y listo..